<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkLabelSetWidgetControls</class>
 <widget class="QWidget" name="QmitkLabelSetWidgetControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>176</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="m_lblCaption">
     <property name="text">
      <string>Caption</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QmitkLabelSetTableWidget" name="m_LabelSetTableWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QToolButton" name="m_btDeactivateTool">
       <property name="toolTip">
        <string>Disable the active tool</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/SegmentationUI.qrc">
         <normaloff>:/Qmitk/SegmentationInteractor_48x48.png</normaloff>:/Qmitk/SegmentationInteractor_48x48.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_btLockExterior">
       <property name="toolTip">
        <string>Lock/Unlock exterior</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/SegmentationUI.qrc">
         <normaloff>:/Qmitk/UnlockExterior_48x48.png</normaloff>
         <normalon>:/Qmitk/LockExterior_48x48.png</normalon>:/Qmitk/UnlockExterior_48x48.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_btAddLayer">
       <property name="toolTip">
        <string>Add a layer to the current segmentation session</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/SegmentationUI.qrc">
         <normaloff>:/Qmitk/AddLayer_48x48.png</normaloff>:/Qmitk/AddLayer_48x48.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_btDeleteLayer">
       <property name="toolTip">
        <string>Delete the active layer</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/SegmentationUI.qrc">
         <normaloff>:/Qmitk/DeleteLayer_48x48.png</normaloff>:/Qmitk/DeleteLayer_48x48.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_btPreviousLayer">
       <property name="toolTip">
        <string>Change to the previous available layer</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/SegmentationUI.qrc">
         <normaloff>:/Qmitk/PreviousLayer_48x48.png</normaloff>:/Qmitk/PreviousLayer_48x48.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_cbActiveLayer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Switch to a layer</string>
       </property>
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_btNextLayer">
       <property name="toolTip">
        <string>Change to the next available layer</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/SegmentationUI.qrc">
         <normaloff>:/Qmitk/NextLayer_48x48.png</normaloff>:/Qmitk/NextLayer_48x48.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ctkSearchBox" name="m_LabelSearchBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkLabelSetTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">QmitkLabelSetTableWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSearchBox</class>
   <extends>QLineEdit</extends>
   <header location="global">ctkSearchBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/SegmentationUI.qrc"/>
 </resources>
 <connections/>
</ui>
